### PR TITLE
Migrate transportation tool to layout-side-nav template to remove breadcrumb template

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Migrate transportation tool to layout-side-nav template to remove breadcrumb template.

## How to test this PR

1. Visit http://localhost:8000/consumer-tools/educator-tools/resources-youth-employment-programs/transportation-tool/ and compare to https://www.consumerfinance.gov/consumer-tools/educator-tools/resources-youth-employment-programs/transportation-tool/

## Notes and todos

- Pay particular close attention to the notifications, which moved in the markup. Originally they were just inside the `</main>` now they are a few divs in due to the structure of the base template.

## Screenshot

Before:

Screen Shot 2021-04-14 at 5 30 50 PM

After:

Screen Shot 2021-04-14 at 5 30 58 PM

### CODE REVIEW

File: cfgov/jinja2/v1/youth_employment_success/index.html [hunk 1]
```diff
@@ -1,62 +1,75 @@
...
-{% block content %}
+{% block pre_content %}
+    <div class="content_wrapper">
+        <nav class="breadcrumbs" aria-label="Breadcrumbs">
+            {{ svg_icon('left') }}
+            <a class="breadcrumbs_link" href="/consumer-tools/educator-tools/">
+              Educator Tools
+            </a>
+            <a class="breadcrumbs_link" href="/consumer-tools/educator-tools/resources-youth-employment-programs/">
+              Resources for Youth Employment Programs
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/youth_employment_success/index.html:28]: This doesn't appear to have saved much over having the template, but several pages use a block like this and if we refactor the base template they use, this block can go for all of them.

Comment (cwdavies) [reply cfgov/jinja2/v1/youth_employment_success/index.html:28]: Let me know if and when the base template is refactored

**Review by cwdavies — APPROVED** [review]

File: cfgov/jinja2/v1/youth_employment_success/side-nav.html [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
-                      m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/buying-a-car/">
-                Buying a car
-            </a>
-          </li>
-        </ul>    
-      </div>
+      <ul class="m-list
+                m-list__unstyled
+                o-secondary-navigation_list
+                o-secondary-navigation_list__parents">
```
Comment (Scotchester) [inline cfgov/jinja2/v1/youth_employment_success/side-nav.html:7]:
```suggestion
      <ul class="m-list
                 m-list__unstyled
                 o-secondary-navigation_list
                 o-secondary-navigation_list__parents">
```

File: cfgov/jinja2/v1/youth_employment_success/side-nav.html [hunk 3]
```diff
@@ -1,38 +1,36 @@
...
+                    m-nav-link__parent" href="/consumer-tools/educator-tools/resources-youth-employment-programs/buying-a-car/">
+              Buying a car
+          </a>
+        </li>
+      </ul>
     </div>
-  </nav>
-</aside>
+  </div>
+</nav>
```
Comment (Scotchester) [inline cfgov/jinja2/v1/youth_employment_success/side-nav.html:36]: Trivial: Missing EOF newline.

**Review by Scotchester — APPROVED** [review]
This Looks good to me. One broader suggestion, though: since we're already creating a diff on almost the entirety of both files, could we go ahead and standardize them on our preferred four-space indentation?

File: cfgov/jinja2/v1/youth_employment_success/side-nav.html (hunk 3 above)
Comment (anselmbradford) [reply cfgov/jinja2/v1/youth_employment_success/side-nav.html]: Hm pulling up the file, the EOF newline is there.

Comment (Scotchester) [reply cfgov/jinja2/v1/youth_employment_success/side-nav.html]: It was added in your latest commit. Did you commit that via the GitHub suggestion mechanism, to fix the class alignment? Maybe it did it automatically when you did.

Comment (anselmbradford) [reply cfgov/jinja2/v1/youth_employment_success/side-nav.html]: Hm not sure what happened, but on second check I saw and fixed it.

**Review by virginiacc — APPROVED** [review]

**Pull request merged**